qemu.initd

```diff
@@ -205,7 +205,7 @@ version() {
 #-------------------------------- Helpers -------------------------------
 
 is_running() {
-	[ -e "$pidfile" ] && kill -0 "$(cat "$pidfile")"
+	[ -e "$pidfile" ] && kill -0 "$(cat "$pidfile")" &> /dev/null
```

Comment:
This is bashism, POSIX does not define `&>`; `>/dev/null 2>&1` should be used instead.